addOns/graphql/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this add-on will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+### Added
+- Support for relative file paths.
```

Comment:
"and those including variables"?  doesn't apply here (something like that was in earlier entries)

Reply:
In this case they were already supported.

Reply:
Oh okay.